corehq/apps/accounting/migrations/0032_customerinvoice.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
```

Comment:
It's good practice to squash multiple migrations in the same PR into one migration.  You can do this manually or use django's tools to help:
https://docs.djangoproject.com/en/1.11/topics/migrations/#migration-squashing
https://docs.djangoproject.com/en/1.11/ref/django-admin/#django-admin-squashmigrations

Reply:
At what point can I delete the old migration files? Once this squashed migrations file has been deployed to everything?

Reply:
Another point: when you do delete the squashed migrations, you should also remove the ```replaces``` field from the new migration at the same time.

Reply:
So, now that I have that squashed migrations file, I should run migrations -> delete `replaces` -> run migrations again?

I haven't put this on staging, so I'm just going to do all this locally.

Reply:
👍